src/config/tokens/core/_core-typography.scss

```diff
@@ -11,10 +11,10 @@ $title-font-family: $font-family-english-heading, $font-family-other-languages;
 // Font definition shorthands per semantics atom definitions
 $font-h1-bold: 700 32px/40px $title-font-family;
 $font-h1-normal: 500 32px/40px $title-font-family;
-$font-h1-light: 300 24px/40px $title-font-family;
+$font-h1-light: 300 32px/40px $title-font-family;
 $font-h2-bold: 700 24px/30px $title-font-family;
 $font-h2-normal: 500 24px/30px $title-font-family;
-$font-h2-light: 300 18px/30px $title-font-family;
+$font-h2-light: 300 24px/30px $title-font-family;
```

Comment:
Is it only affecting the new font definitions?

Reply:
yes